src/actions.js

```diff
@@ -122,8 +122,10 @@ export function fetchInsuree(mm, chfid) {
   return graphql(payload, "INSUREE_INSUREE");
 }
 
-export function fetchInsureeFull(mm, uuid) {
-  let payload = formatPageQuery("insurees", [`uuid:"${uuid}"`], INSUREE_FULL_PROJECTION(mm), "clientMutationId");
+export function fetchInsureeFull(mm, uuid, ignoreLocation=false) {
+  let args = [`uuid:"${uuid}"`]
```

Comment:
you add add here in two lines missing semicolons

Reply:
done